WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -0,0 +1,104 @@
+import UIKit
+
+final class CompliancePopoverCoordinator {
+    fileprivate enum Constants {
+        static let shouldShowCompliancePopup = "shouldShowCompliancePopup"
+    }
+
+    private let viewController: UIViewController
+    private let complianceService = ComplianceLocationService()
+    private let defaults: UserDefaults
+
+    init(viewController: UIViewController, defaults: UserDefaults = UserDefaults.standard) {
+        self.viewController = viewController
+        self.defaults = defaults
+    }
+
+    func presentIfNeeded() {
+        guard FeatureFlag.compliancePopover.enabled else {
+            return
+        }
+        complianceService.getIPCountryCode { [weak self] result in
+            switch result {
+            case .success(let countryCode):
+                guard let self, self.shouldShowPrivacyBanner(countryCode: countryCode) else {
+                    return
+                }
+                DispatchQueue.main.async {
+                    let complianceViewModel = CompliancePopoverViewModel(defaults: self.defaults)
+                    complianceViewModel.coordinator = self
+                    let complianceViewController = CompliancePopoverViewController(viewModel: complianceViewModel)
+                    let bottomSheetViewController = BottomSheetViewController(childViewController: complianceViewController, customHeaderSpacing: 0)
+
+                    bottomSheetViewController.show(from: self.viewController)
+                }
+            case .failure(let error):
+                ()
+            }
+        }
+    }
+
+    func shouldShowPrivacyBanner(countryCode: String) -> Bool {
+        let isCountryInEU = Self.gdprCountryCodes.contains(countryCode)
+        return isCountryInEU && defaults.shouldShowCompliancePopup
```

Comment:
[Blocker]
I wasn't able to get the popover to show and I believe this is the culprit.

The default boolean value is false, which means `shouldShowCompliancePopup` is initially false. And I see that we don't set it to true anywhere. My understanding is that `shouldShowPrivacyBanner` will always return false, unless I'm missing something.

Reply:
I suggest changing the user defaults flag to `didShowCompliancePopup` and setting it to true when the popover is displayed.

```suggestion
        return isCountryInEU && !defaults.didShowCompliancePopup
```

Reply:
Ah you're right. I had flipped the name of it and not the value. Thanks for catching it